internal/runnerinstall/k8s.go

```diff
@@ -3,7 +3,7 @@ package runnerinstall
 import (
 	"context"
 	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
-	"github.com/hashicorp/waypoint/internal/installutil/helm"
+	installutil "github.com/hashicorp/waypoint/internal/installutil/helm"
```

Comment:
I can take care of this within the `uninstall` implementation so this is more a question for me I suppose, but should we include helm within the `installutil` import here?
I only mention this because we will also likely need to import a `k8s` specific installutil as well which will help DRY up the similarities between server and runner uninstallation implementations regarding persistent volumes whose cleanup is not managed by helm.

Reply:
Good point! I hadn't yet added such an `installutil` for non-Helm k8s stuff specifically, so I will leave this as-is for this PR, but for the alias of that import in other places I will phase in specifying which "installutil" it is, since there could be multiple.